geoutils/georaster.py

```diff
@@ -203,6 +203,20 @@ def __str__(self):
         """ Provide string of information about Raster. """
         return self.info()
 
+    def __eq__(self, other) -> bool:
```

Comment:
so this would mean the data + CRS are equal, but not necessarily on disk (could be a copy) ?!

Reply:
Nvm I see this in your description now